evm_arithmetization/src/cpu/kernel/asm/mpt/linked_list/linked_list.asm

```diff
@@ -24,8 +24,8 @@
 /// The values at the respective positions are:
 /// - 0: The account key
 /// - 1: The slot key
-/// - 2: A ptr to the payload (the stored value)
-/// - 3: A ptr to the initial payload.
+/// - 2: The slot value.
```

Comment:
Unrelated to the PR, but it wasn't changed in https://github.com/0xPolygonZero/zk_evm/pull/433.